CHANGELOG.md

```diff
@@ -5,15 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Ivanchuk Beta1
+## Hammer-10
+
+### Fixed
+- Don't set User.current_user [(#293)](https://github.com/ManageIQ/manageiq-automation_engine/pull/293)
```

Comment:
@JPrause I feel a more accurate title for this PR would be `Replace "User.current_user" with "User.with_user"` 

@tinaafitz Are you ok with this?  Should we change the PR title as well?

Reply:
Sorry @JPrause I wanted to title to read as the whole string below:

`Replace User.current_user with User.with_user`

I removed the quotes this time.  Also updated the PR to match.

Reply:
No worries,...I guess a literal guy. Ha! Will update now.

Reply:
and I went over to the PR to make sure I got it right. :smile:

Reply:
I wondering if I did that with you in mind.  🤔 

Thanks for the updates @JPrause

Reply:
@gmcculloug yes, that makes sense. I saw that PR title was updated.